python/paddle/tensor/math.py

```diff
@@ -562,34 +562,52 @@ def remainder(x, y, name=None):
 
 def multiply(x, y, axis=-1, name=None):
     """
-	:alias_main: paddle.multiply
-	:alias: paddle.multiply,paddle.tensor.multiply,paddle.tensor.math.multiply
+    multiply two tensors element-wise. The equation is:
 
-Examples:
+    .. math::
+        out = x * y
 
-    .. code-block:: python
+    **Note**:
+    ``paddle.multiply`` supports broadcasting. If you want know more about broadcasting, please refer to :ref:`user_guide_broadcasting` .
 
-        import paddle
-        import numpy as np
+    Args:
+        x (Tensor): the input tensor, it's data type should be float32, float64, int32, int64.
```

Comment:
English grammar: change "it's" to "its"

"it's" means "it is" or "it has". Here you are saying the data type belonging to the input tensor, you should use its.